### PR TITLE
feat: dark mode support for memo detail

### PR DESCRIPTION
**Pull request by hyoban**
part of #521

### CODE REVIEW

File: web/src/css/tailwind.css [hunk 1]
```diff
@@ -14,22 +14,28 @@
...
+    @apply btn-normal text-gray-600 border-none hover:shadow-none;
+  }
+  
+  .input-text {
+    @apply w-full px-3 py-2 leading-6 text-sm border rounded;
+  }
 
-.input-text {
-  @apply w-full px-3 py-2 leading-6 text-sm border rounded;
+  .card-memo {
```
Comment (boojack) [inline web/src/css/tailwind.css:38]: As it's used in only two places, it doesn't look like a common util class. 

https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction

**Review by boojack — COMMENTED** [review]
LGTM

File: web/src/css/tailwind.css (hunk 1 above)
Comment (hyoban) [reply web/src/css/tailwind.css:38]: But it looks like both need to be styled the same way, and also the cards in the home page are styled the same

Comment (boojack) [reply web/src/css/tailwind.css:38]: We can create a less/module.css file including the same layout css codes. Then import it in the component files. 

The tailwind.css should save the basic class definitions which will be used in multiple files.

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**